### ISSUE TITLE
Version 1.1.9 for CRAN

### ISSUE THREAD

**Issue by mebrooks** [glmmTMB/glmmTMB#1000]
CRAN emailed me and requested a new version by March 17th. I was hoping to get the `dispRE` branch ready for that, but think it might be time to start working towards this in parallel. I have to teach today, so I probably won't get to it until tomorrow.

We have [issues to fix](https://cran.r-project.org/web/checks/check_results_glmmTMB.html). The email specifically said to remember to look at the 'Additional issues'.

**Comment by bbolker** [comment]
Sigh, the stuff on M1Mac is that old platform-variable test for doing the right thing when the Hessian can't be inverted.  In fact we had already commented it out in cd020c22940bbd8 (I think it would be better to `skip()` it but anyway ...

```
 ── 1. Failure ('test-methods.R:656:9'): bad inversion in vcov ──────────────────
  all(is.na(vcov(m)$cond)) is not TRUE
  
  `actual`:   FALSE
  `expected`: TRUE 
```

The other issues are pretty trivial, I may already have fixed them in some branch or other ...

I, too, have a lot of stuff on branches that's not merged into devel yet (especially `check_negbinom_vals`, `priors`, `extend_re_form`) ...

**Comment by mebrooks** [comment]
They contacted me a while ago on March 3rd when I was on vacation, so it slipped to the back of my mind. Sorry about that. I can focus on this today and over the weekend.

**Comment by mebrooks** [comment]
`check --as-cran` seems OK so far except for this
```
* checking HTML version of manual ... NOTE
Found the following HTML validation problems:
Owls.html:4:1 (Owls.Rd:2): Warning: <link> inserting "type" attribute
Owls.html:12:1 (Owls.Rd:2): Warning: <script> proprietary attribute "onload"
Owls.html:12:1 (Owls.Rd:2): Warning: <script> inserting "type" attribute
Owls.html:17:1 (Owls.Rd:2): Warning: <table> lacks "summary" attribute
RHSForm.html:4:1 (RHSForm.Rd:5): Warning: <link> inserting "type" attribute
RHSForm.html:12:1 (RHSForm.Rd:5): Warning: <script> proprietary attribute "onload"
RHSForm.html:12:1 (RHSForm.Rd:5): Warning: <script> inserting "type" attribute
RHSForm.html:17:1 (RHSForm.Rd:5): Warning: <table> lacks "summary" attribute
RHSForm.html:35:1 (RHSForm.Rd:10): Warning: <table> lacks "summary" attribute
Salamanders.html:4:1 (Salamanders.Rd:2): Warning: <link> inserting "type" attribute
Salamanders.html:12:1 (Salamanders.Rd:2): Warning: <script> proprietary attribute "onload"
Salamanders.html:12:1 (Salamanders.Rd:2): Warning: <script> inserting "type" attribute
Salamanders.html:17:1 (Salamanders.Rd:2): Warning: <table> lacks "summary" attribute
VarCorr.glmmTMB.html:4:1 (VarCorr.glmmTMB.Rd:6): Warning: <link> inserting "type" attribute
VarCorr.glmmTMB.html:12:1 (VarCorr.glmmTMB.Rd:6): Warning: <script> proprietary attribute "onload"
VarCorr.glmmTMB.html:12:1 (VarCorr.glmmTMB.Rd:6): Warning: <script> inserting "type" attribute
VarCorr.glmmTMB.html:17:1 (VarCorr.glmmTMB.Rd:6): Warning: <table> lacks "summary" attribute
VarCorr.glmmTMB.html:36:1 (VarCorr.glmmTMB.Rd:11): Warning: <table> lacks "summary" attribute
bootmer_methods.html:4:1 (bootmer_methods.Rd:6): Warning: <link> inserting "type" attribute
bootmer_methods.html:12:1 (bootmer_methods.Rd:6): Warning: <script> proprietary attribute "onload"
bootmer_methods.html:12:1 (bootmer_methods.Rd:6): Warning: <script> inserting "type" attribute
bootmer_methods.html:17:1 (bootmer_methods.Rd:6): Warning: <table> lacks "summary" attribute
bootmer_methods.html:39:1 (bootmer_methods.Rd:13): Warning: <table> lacks "summary" attribute
checkDepPackageVersion.html:4:1 (checkDepPackageVersion.Rd:5): Warning: <link> inserting "type" attribute
checkDepPackageVersion.html:12:1 (checkDepPackageVersion.Rd:5): Warning: <script> proprietary attribute "onload"
checkDepPackageVersion.html:12:1 (checkDepPackageVersion.Rd:5): Warning: <script> inserting "type" attribute
checkDepPackageVersion.html:17:1 (checkDepPackageVersion.Rd:5): Warning: <table> lacks "summary" attribute
checkDepPackageVersion.html:40:1 (checkDepPackageVersion.Rd:15): Warning: <table> lacks "summary" attribute
confint.glmmTMB.html:4:1 (confint.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
confint.glmmTMB.html:12:1 (confint.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
confint.glmmTMB.html:12:1 (confint.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
confint.glmmTMB.html:17:1 (confint.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
confint.glmmTMB.html:49:1 (confint.glmmTMB.Rd:23): Warning: <table> lacks "summary" attribute
diagnose.html:4:1 (diagnose.Rd:5): Warning: <link> inserting "type" attribute
diagnose.html:12:1 (diagnose.Rd:5): Warning: <script> proprietary attribute "onload"
diagnose.html:12:1 (diagnose.Rd:5): Warning: <script> inserting "type" attribute
diagnose.html:17:1 (diagnose.Rd:5): Warning: <table> lacks "summary" attribute
diagnose.html:53:1 (diagnose.Rd:22): Warning: <table> lacks "summary" attribute
dot-adjustX.html:4:1 (dot-adjustX.Rd:5): Warning: <link> inserting "type" attribute
dot-adjustX.html:12:1 (dot-adjustX.Rd:5): Warning: <script> proprietary attribute "onload"
dot-adjustX.html:12:1 (dot-adjustX.Rd:5): Warning: <script> inserting "type" attribute
dot-adjustX.html:17:1 (dot-adjustX.Rd:5): Warning: <table> lacks "summary" attribute
dot-adjustX.html:19:1 (dot-adjustX.Rd:5): Warning: <h2> attribute "id" has invalid value ".adjustX"
dot-adjustX.html:39:1 (dot-adjustX.Rd:12): Warning: <table> lacks "summary" attribute
dot-adjustX.html:40:14 (dot-adjustX.Rd:12): Warning: <code> attribute "id" has invalid value ".adjustX_:_X"
dot-adjustX.html:44:14 (dot-adjustX.Rd:14): Warning: <code> attribute "id" has invalid value ".adjustX_:_tol"
dot-adjustX.html:48:14 (dot-adjustX.Rd:16): Warning: <code> attribute "id" has invalid value ".adjustX_:_why_dropped"
dot-checkRankX.html:4:1 (dot-checkRankX.Rd:5): Warning: <link> inserting "type" attribute
dot-checkRankX.html:12:1 (dot-checkRankX.Rd:5): Warning: <script> proprietary attribute "onload"
dot-checkRankX.html:12:1 (dot-checkRankX.Rd:5): Warning: <script> inserting "type" attribute
dot-checkRankX.html:17:1 (dot-checkRankX.Rd:5): Warning: <table> lacks "summary" attribute
dot-checkRankX.html:19:1 (dot-checkRankX.Rd:5): Warning: <h2> attribute "id" has invalid value ".checkRankX"
dot-collectDuplicates.html:4:1 (dot-collectDuplicates.Rd:5): Warning: <link> inserting "type" attribute
dot-collectDuplicates.html:12:1 (dot-collectDuplicates.Rd:5): Warning: <script> proprietary attribute "onload"
dot-collectDuplicates.html:12:1 (dot-collectDuplicates.Rd:5): Warning: <script> inserting "type" attribute
dot-collectDuplicates.html:17:1 (dot-collectDuplicates.Rd:5): Warning: <table> lacks "summary" attribute
dot-collectDuplicates.html:19:1 (dot-collectDuplicates.Rd:5): Warning: <h2> attribute "id" has invalid value ".collectDuplicates"
downstream_methods.html:4:1 (downstream_methods.Rd:8): Warning: <link> inserting "type" attribute
downstream_methods.html:12:1 (downstream_methods.Rd:8): Warning: <script> proprietary attribute "onload"
downstream_methods.html:12:1 (downstream_methods.Rd:8): Warning: <script> inserting "type" attribute
downstream_methods.html:17:1 (downstream_methods.Rd:8): Warning: <table> lacks "summary" attribute
downstream_methods.html:77:1 (downstream_methods.Rd:24): Warning: <table> lacks "summary" attribute
dtruncated_nbinom2.html:4:1 (dtruncated_nbinom2.Rd:7): Warning: <link> inserting "type" attribute
dtruncated_nbinom2.html:12:1 (dtruncated_nbinom2.Rd:7): Warning: <script> proprietary attribute "onload"
dtruncated_nbinom2.html:12:1 (dtruncated_nbinom2.Rd:7): Warning: <script> inserting "type" attribute
dtruncated_nbinom2.html:17:1 (dtruncated_nbinom2.Rd:7): Warning: <table> lacks "summary" attribute
dtruncated_nbinom2.html:39:1 (dtruncated_nbinom2.Rd:16): Warning: <table> lacks "summary" attribute
epil2.html:4:1 (epil2.Rd:2): Warning: <link> inserting "type" attribute
epil2.html:12:1 (epil2.Rd:2): Warning: <script> proprietary attribute "onload"
epil2.html:12:1 (epil2.Rd:2): Warning: <script> inserting "type" attribute
epil2.html:17:1 (epil2.Rd:2): Warning: <table> lacks "summary" attribute
expandGrpVar.html:4:1 (expandGrpVar.Rd:5): Warning: <link> inserting "type" attribute
expandGrpVar.html:12:1 (expandGrpVar.Rd:5): Warning: <script> proprietary attribute "onload"
expandGrpVar.html:12:1 (expandGrpVar.Rd:5): Warning: <script> inserting "type" attribute
expandGrpVar.html:17:1 (expandGrpVar.Rd:5): Warning: <table> lacks "summary" attribute
expandGrpVar.html:35:1 (expandGrpVar.Rd:10): Warning: <table> lacks "summary" attribute
family_params.html:4:1 (family_params.Rd:5): Warning: <link> inserting "type" attribute
family_params.html:12:1 (family_params.Rd:5): Warning: <script> proprietary attribute "onload"
family_params.html:12:1 (family_params.Rd:5): Warning: <script> inserting "type" attribute
family_params.html:17:1 (family_params.Rd:5): Warning: <table> lacks "summary" attribute
family_params.html:37:1 (family_params.Rd:10): Warning: <table> lacks "summary" attribute
findReTrmClasses.html:4:1 (findReTrmClasses.Rd:5): Warning: <link> inserting "type" attribute
findReTrmClasses.html:12:1 (findReTrmClasses.Rd:5): Warning: <script> proprietary attribute "onload"
findReTrmClasses.html:12:1 (findReTrmClasses.Rd:5): Warning: <script> inserting "type" attribute
findReTrmClasses.html:17:1 (findReTrmClasses.Rd:5): Warning: <table> lacks "summary" attribute
fitTMB.html:4:1 (fitTMB.Rd:6): Warning: <link> inserting "type" attribute
fitTMB.html:12:1 (fitTMB.Rd:6): Warning: <script> proprietary attribute "onload"
fitTMB.html:12:1 (fitTMB.Rd:6): Warning: <script> inserting "type" attribute
fitTMB.html:17:1 (fitTMB.Rd:6): Warning: <table> lacks "summary" attribute
fitTMB.html:44:1 (fitTMB.Rd:13): Warning: <table> lacks "summary" attribute
fixef.html:4:1 (fixef.Rd:7): Warning: <link> inserting "type" attribute
fixef.html:12:1 (fixef.Rd:7): Warning: <script> proprietary attribute "onload"
fixef.html:12:1 (fixef.Rd:7): Warning: <script> inserting "type" attribute
fixef.html:17:1 (fixef.Rd:7): Warning: <table> lacks "summary" attribute
fixef.html:36:1 (fixef.Rd:12): Warning: <table> lacks "summary" attribute
formatVC.html:4:1 (formatVC.Rd:5): Warning: <link> inserting "type" attribute
formatVC.html:12:1 (formatVC.Rd:5): Warning: <script> proprietary attribute "onload"
formatVC.html:12:1 (formatVC.Rd:5): Warning: <script> inserting "type" attribute
formatVC.html:17:1 (formatVC.Rd:5): Warning: <table> lacks "summary" attribute
formatVC.html:43:1 (formatVC.Rd:17): Warning: <table> lacks "summary" attribute
formfuns.html:4:1 (formfuns.Rd:18): Warning: <link> inserting "type" attribute
formfuns.html:12:1 (formfuns.Rd:18): Warning: <script> proprietary attribute "onload"
formfuns.html:12:1 (formfuns.Rd:18): Warning: <script> inserting "type" attribute
formfuns.html:17:1 (formfuns.Rd:18): Warning: <table> lacks "summary" attribute
formfuns.html:71:1 (formfuns.Rd:56): Warning: <table> lacks "summary" attribute
formula.glmmTMB.html:4:1 (formula.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
formula.glmmTMB.html:12:1 (formula.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
formula.glmmTMB.html:12:1 (formula.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
formula.glmmTMB.html:17:1 (formula.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
formula.glmmTMB.html:36:1 (formula.glmmTMB.Rd:10): Warning: <table> lacks "summary" attribute
getCapabilities.html:4:1 (getCapabilities.Rd:5): Warning: <link> inserting "type" attribute
getCapabilities.html:12:1 (getCapabilities.Rd:5): Warning: <script> proprietary attribute "onload"
getCapabilities.html:12:1 (getCapabilities.Rd:5): Warning: <script> inserting "type" attribute
getCapabilities.html:17:1 (getCapabilities.Rd:5): Warning: <table> lacks "summary" attribute
getCapabilities.html:35:1 (getCapabilities.Rd:10): Warning: <table> lacks "summary" attribute
getGrpVar.html:4:1 (getGrpVar.Rd:5): Warning: <link> inserting "type" attribute
getGrpVar.html:12:1 (getGrpVar.Rd:5): Warning: <script> proprietary attribute "onload"
getGrpVar.html:12:1 (getGrpVar.Rd:5): Warning: <script> inserting "type" attribute
getGrpVar.html:17:1 (getGrpVar.Rd:5): Warning: <table> lacks "summary" attribute
getGrpVar.html:35:1 (getGrpVar.Rd:10): Warning: <table> lacks "summary" attribute
getME.glmmTMB.html:4:1 (getME.glmmTMB.Rd:6): Warning: <link> inserting "type" attribute
getME.glmmTMB.html:12:1 (getME.glmmTMB.Rd:6): Warning: <script> proprietary attribute "onload"
getME.glmmTMB.html:12:1 (getME.glmmTMB.Rd:6): Warning: <script> inserting "type" attribute
getME.glmmTMB.html:17:1 (getME.glmmTMB.Rd:6): Warning: <table> lacks "summary" attribute
getME.glmmTMB.html:40:1 (getME.glmmTMB.Rd:15): Warning: <table> lacks "summary" attribute
getReStruc.html:4:1 (getReStruc.Rd:5): Warning: <link> inserting "type" attribute
getReStruc.html:12:1 (getReStruc.Rd:5): Warning: <script> proprietary attribute "onload"
getReStruc.html:12:1 (getReStruc.Rd:5): Warning: <script> inserting "type" attribute
getReStruc.html:17:1 (getReStruc.Rd:5): Warning: <table> lacks "summary" attribute
getReStruc.html:39:1 (getReStruc.Rd:12): Warning: <table> lacks "summary" attribute
getReStruc.html:70:1 (getReStruc.Rd:27): Warning: <table> lacks "summary" attribute
getXReTrms.html:4:1 (getXReTrms.Rd:5): Warning: <link> inserting "type" attribute
getXReTrms.html:12:1 (getXReTrms.Rd:5): Warning: <script> proprietary attribute "onload"
getXReTrms.html:12:1 (getXReTrms.Rd:5): Warning: <script> inserting "type" attribute
getXReTrms.html:17:1 (getXReTrms.Rd:5): Warning: <table> lacks "summary" attribute
getXReTrms.html:44:1 (getXReTrms.Rd:19): Warning: <table> lacks "summary" attribute
getXReTrms.html:84:1 (getXReTrms.Rd:37): Warning: <table> lacks "summary" attribute
get_cor.html:4:1 (get_cor.Rd:6): Warning: <link> inserting "type" attribute
get_cor.html:12:1 (get_cor.Rd:6): Warning: <script> proprietary attribute "onload"
get_cor.html:12:1 (get_cor.Rd:6): Warning: <script> inserting "type" attribute
get_cor.html:17:1 (get_cor.Rd:6): Warning: <table> lacks "summary" attribute
get_cor.html:37:1 (get_cor.Rd:13): Warning: <table> lacks "summary" attribute
glmmTMB.html:209:199 (glmmTMB.Rd:162): Warning: replacing invalid character code 128
glmmTMB.html:209:200 (glmmTMB.Rd:162): Warning: replacing invalid character code 147
glmmTMB.html:4:1 (glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
glmmTMB.html:12:1 (glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
glmmTMB.html:12:1 (glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
glmmTMB.html:17:1 (glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
glmmTMB.html:53:1 (glmmTMB.Rd:28): Warning: <table> lacks "summary" attribute
glmmTMBControl.html:4:1 (glmmTMBControl.Rd:5): Warning: <link> inserting "type" attribute
glmmTMBControl.html:12:1 (glmmTMBControl.Rd:5): Warning: <script> proprietary attribute "onload"
glmmTMBControl.html:12:1 (glmmTMBControl.Rd:5): Warning: <script> inserting "type" attribute
glmmTMBControl.html:17:1 (glmmTMBControl.Rd:5): Warning: <table> lacks "summary" attribute
glmmTMBControl.html:47:1 (glmmTMBControl.Rd:22): Warning: <table> lacks "summary" attribute
glmmTMB_methods.html:4:1 (glmmTMB_methods.Rd:6): Warning: <link> inserting "type" attribute
glmmTMB_methods.html:12:1 (glmmTMB_methods.Rd:6): Warning: <script> proprietary attribute "onload"
glmmTMB_methods.html:12:1 (glmmTMB_methods.Rd:6): Warning: <script> inserting "type" attribute
glmmTMB_methods.html:17:1 (glmmTMB_methods.Rd:6): Warning: <table> lacks "summary" attribute
glmmTMB_methods.html:45:1 (glmmTMB_methods.Rd:19): Warning: <table> lacks "summary" attribute
gt_load.html:4:1 (gt_load.Rd:6): Warning: <link> inserting "type" attribute
gt_load.html:12:1 (gt_load.Rd:6): Warning: <script> proprietary attribute "onload"
gt_load.html:12:1 (gt_load.Rd:6): Warning: <script> inserting "type" attribute
gt_load.html:17:1 (gt_load.Rd:6): Warning: <table> lacks "summary" attribute
gt_load.html:39:1 (gt_load.Rd:13): Warning: <table> lacks "summary" attribute
mkTMBStruc.html:4:1 (mkTMBStruc.Rd:5): Warning: <link> inserting "type" attribute
mkTMBStruc.html:12:1 (mkTMBStruc.Rd:5): Warning: <script> proprietary attribute "onload"
mkTMBStruc.html:12:1 (mkTMBStruc.Rd:5): Warning: <script> inserting "type" attribute
mkTMBStruc.html:17:1 (mkTMBStruc.Rd:5): Warning: <table> lacks "summary" attribute
mkTMBStruc.html:59:1 (mkTMBStruc.Rd:34): Warning: <table> lacks "summary" attribute
nbinom2.html:141:153 (nbinom2.Rd:94): Warning: replacing invalid character code 128
nbinom2.html:141:154 (nbinom2.Rd:94): Warning: replacing invalid character code 147
nbinom2.html:150:62 (nbinom2.Rd:97): Warning: replacing invalid character code 128
nbinom2.html:150:63 (nbinom2.Rd:97): Warning: replacing invalid character code 147
nbinom2.html:150:72 (nbinom2.Rd:97): Warning: replacing invalid character code 128
nbinom2.html:150:73 (nbinom2.Rd:97): Warning: replacing invalid character code 147
nbinom2.html:153:213 (nbinom2.Rd:98): Warning: replacing invalid character code 128
nbinom2.html:153:214 (nbinom2.Rd:98): Warning: replacing invalid character code 147
nbinom2.html:165:156 (nbinom2.Rd:102): Warning: replacing invalid character code 128
nbinom2.html:165:157 (nbinom2.Rd:102): Warning: replacing invalid character code 147
nbinom2.html:168:160 (nbinom2.Rd:103): Warning: replacing invalid character code 128
nbinom2.html:168:161 (nbinom2.Rd:103): Warning: replacing invalid character code 147
nbinom2.html:4:1 (nbinom2.Rd:21): Warning: <link> inserting "type" attribute
nbinom2.html:12:1 (nbinom2.Rd:21): Warning: <script> proprietary attribute "onload"
nbinom2.html:12:1 (nbinom2.Rd:21): Warning: <script> inserting "type" attribute
nbinom2.html:17:1 (nbinom2.Rd:21): Warning: <table> lacks "summary" attribute
nbinom2.html:65:1 (nbinom2.Rd:56): Warning: <table> lacks "summary" attribute
nbinom2.html:118:1 (nbinom2.Rd:60): Warning: <table> lacks "summary" attribute
numFactor.html:4:1 (numFactor.Rd:6): Warning: <link> inserting "type" attribute
numFactor.html:12:1 (numFactor.Rd:6): Warning: <script> proprietary attribute "onload"
numFactor.html:12:1 (numFactor.Rd:6): Warning: <script> inserting "type" attribute
numFactor.html:17:1 (numFactor.Rd:6): Warning: <table> lacks "summary" attribute
numFactor.html:37:1 (numFactor.Rd:13): Warning: <table> lacks "summary" attribute
omp_check.html:4:1 (omp_check.Rd:5): Warning: <link> inserting "type" attribute
omp_check.html:12:1 (omp_check.Rd:5): Warning: <script> proprietary attribute "onload"
omp_check.html:12:1 (omp_check.Rd:5): Warning: <script> inserting "type" attribute
omp_check.html:17:1 (omp_check.Rd:5): Warning: <table> lacks "summary" attribute
predict.glmmTMB.html:4:1 (predict.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
predict.glmmTMB.html:12:1 (predict.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
predict.glmmTMB.html:12:1 (predict.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
predict.glmmTMB.html:17:1 (predict.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
predict.glmmTMB.html:50:1 (predict.glmmTMB.Rd:24): Warning: <table> lacks "summary" attribute
print.VarCorr.glmmTMB.html:4:1 (print.VarCorr.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
print.VarCorr.glmmTMB.html:12:1 (print.VarCorr.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
print.VarCorr.glmmTMB.html:12:1 (print.VarCorr.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
print.VarCorr.glmmTMB.html:17:1 (print.VarCorr.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
print.VarCorr.glmmTMB.html:42:1 (print.VarCorr.glmmTMB.Rd:16): Warning: <table> lacks "summary" attribute
profile.glmmTMB.html:4:1 (profile.glmmTMB.Rd:6): Warning: <link> inserting "type" attribute
profile.glmmTMB.html:12:1 (profile.glmmTMB.Rd:6): Warning: <script> proprietary attribute "onload"
profile.glmmTMB.html:12:1 (profile.glmmTMB.Rd:6): Warning: <script> inserting "type" attribute
profile.glmmTMB.html:17:1 (profile.glmmTMB.Rd:6): Warning: <table> lacks "summary" attribute
profile.glmmTMB.html:51:1 (profile.glmmTMB.Rd:25): Warning: <table> lacks "summary" attribute
ranef.glmmTMB.html:4:1 (ranef.glmmTMB.Rd:8): Warning: <link> inserting "type" attribute
ranef.glmmTMB.html:12:1 (ranef.glmmTMB.Rd:8): Warning: <script> proprietary attribute "onload"
ranef.glmmTMB.html:12:1 (ranef.glmmTMB.Rd:8): Warning: <script> inserting "type" attribute
ranef.glmmTMB.html:17:1 (ranef.glmmTMB.Rd:8): Warning: <table> lacks "summary" attribute
ranef.glmmTMB.html:43:1 (ranef.glmmTMB.Rd:17): Warning: <table> lacks "summary" attribute
reexports.html:4:1 (reexports.Rd:7): Warning: <link> inserting "type" attribute
reexports.html:12:1 (reexports.Rd:7): Warning: <script> proprietary attribute "onload"
reexports.html:12:1 (reexports.Rd:7): Warning: <script> inserting "type" attribute
reexports.html:17:1 (reexports.Rd:7): Warning: <table> lacks "summary" attribute
reinstalling.html:4:1 (reinstalling.Rd:5): Warning: <link> inserting "type" attribute
reinstalling.html:12:1 (reinstalling.Rd:5): Warning: <script> proprietary attribute "onload"
reinstalling.html:12:1 (reinstalling.Rd:5): Warning: <script> inserting "type" attribute
reinstalling.html:17:1 (reinstalling.Rd:5): Warning: <table> lacks "summary" attribute
residuals.glmmTMB.html:4:1 (residuals.glmmTMB.Rd:6): Warning: <link> inserting "type" attribute
residuals.glmmTMB.html:12:1 (residuals.glmmTMB.Rd:6): Warning: <script> proprietary attribute "onload"
residuals.glmmTMB.html:12:1 (residuals.glmmTMB.Rd:6): Warning: <script> inserting "type" attribute
residuals.glmmTMB.html:17:1 (residuals.glmmTMB.Rd:6): Warning: <table> lacks "summary" attribute
residuals.glmmTMB.html:39:1 (residuals.glmmTMB.Rd:13): Warning: <table> lacks "summary" attribute
sigma.glmmTMB.html:112:152 (sigma.glmmTMB.Rd:66): Warning: replacing invalid character code 128
sigma.glmmTMB.html:112:153 (sigma.glmmTMB.Rd:66): Warning: replacing invalid character code 147
sigma.glmmTMB.html:121:62 (sigma.glmmTMB.Rd:69): Warning: replacing invalid character code 128
sigma.glmmTMB.html:121:63 (sigma.glmmTMB.Rd:69): Warning: replacing invalid character code 147
sigma.glmmTMB.html:121:72 (sigma.glmmTMB.Rd:69): Warning: replacing invalid character code 128
sigma.glmmTMB.html:121:73 (sigma.glmmTMB.Rd:69): Warning: replacing invalid character code 147
sigma.glmmTMB.html:4:1 (sigma.glmmTMB.Rd:6): Warning: <link> inserting "type" attribute
sigma.glmmTMB.html:12:1 (sigma.glmmTMB.Rd:6): Warning: <script> proprietary attribute "onload"
sigma.glmmTMB.html:12:1 (sigma.glmmTMB.Rd:6): Warning: <script> inserting "type" attribute
sigma.glmmTMB.html:17:1 (sigma.glmmTMB.Rd:6): Warning: <table> lacks "summary" attribute
sigma.glmmTMB.html:39:1 (sigma.glmmTMB.Rd:11): Warning: <table> lacks "summary" attribute
simulate.glmmTMB.html:4:1 (simulate.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
simulate.glmmTMB.html:12:1 (simulate.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
simulate.glmmTMB.html:12:1 (simulate.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
simulate.glmmTMB.html:17:1 (simulate.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
simulate.glmmTMB.html:36:1 (simulate.glmmTMB.Rd:10): Warning: <table> lacks "summary" attribute
simulate_new.html:4:1 (simulate_new.Rd:5): Warning: <link> inserting "type" attribute
simulate_new.html:12:1 (simulate_new.Rd:5): Warning: <script> proprietary attribute "onload"
simulate_new.html:12:1 (simulate_new.Rd:5): Warning: <script> inserting "type" attribute
simulate_new.html:17:1 (simulate_new.Rd:5): Warning: <table> lacks "summary" attribute
simulate_new.html:46:1 (simulate_new.Rd:19): Warning: <table> lacks "summary" attribute
splitForm.html:4:1 (splitForm.Rd:6): Warning: <link> inserting "type" attribute
splitForm.html:12:1 (splitForm.Rd:6): Warning: <script> proprietary attribute "onload"
splitForm.html:12:1 (splitForm.Rd:6): Warning: <script> inserting "type" attribute
splitForm.html:17:1 (splitForm.Rd:6): Warning: <table> lacks "summary" attribute
splitForm.html:45:1 (splitForm.Rd:20): Warning: <table> lacks "summary" attribute
startParams.html:4:1 (startParams.Rd:5): Warning: <link> inserting "type" attribute
startParams.html:12:1 (startParams.Rd:5): Warning: <script> proprietary attribute "onload"
startParams.html:12:1 (startParams.Rd:5): Warning: <script> inserting "type" attribute
startParams.html:17:1 (startParams.Rd:5): Warning: <table> lacks "summary" attribute
startParams.html:51:1 (startParams.Rd:26): Warning: <table> lacks "summary" attribute
sub_specials.html:4:1 (sub_specials.Rd:5): Warning: <link> inserting "type" attribute
sub_specials.html:12:1 (sub_specials.Rd:5): Warning: <script> proprietary attribute "onload"
sub_specials.html:12:1 (sub_specials.Rd:5): Warning: <script> inserting "type" attribute
sub_specials.html:17:1 (sub_specials.Rd:5): Warning: <table> lacks "summary" attribute
sub_specials.html:41:1 (sub_specials.Rd:15): Warning: <table> lacks "summary" attribute
vcov.glmmTMB.html:4:1 (vcov.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
vcov.glmmTMB.html:12:1 (vcov.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
vcov.glmmTMB.html:12:1 (vcov.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
vcov.glmmTMB.html:17:1 (vcov.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
vcov.glmmTMB.html:36:1 (vcov.glmmTMB.Rd:10): Warning: <table> lacks "summary" attribute
weights.glmmTMB.html:4:1 (weights.glmmTMB.Rd:5): Warning: <link> inserting "type" attribute
weights.glmmTMB.html:12:1 (weights.glmmTMB.Rd:5): Warning: <script> proprietary attribute "onload"
weights.glmmTMB.html:12:1 (weights.glmmTMB.Rd:5): Warning: <script> inserting "type" attribute
weights.glmmTMB.html:17:1 (weights.glmmTMB.Rd:5): Warning: <table> lacks "summary" attribute
weights.glmmTMB.html:36:1 (weights.glmmTMB.Rd:10): Warning: <table> lacks "summary" attribute
```

Can we ignore that?

**Comment by bbolker** [comment]
https://stat.ethz.ch/pipermail/r-sig-mac/2022-April/014412.html ? (Mac/[HTML tidy](http://binaries.html-tidy.org/) issue)?

**Comment by bbolker** [comment]
- error accessing https://doi.org/10.1111/2041-210X.12236 : false positive. This kind of problem (403 error on a valid URL) usually comes up because the journal web site detects that it's being accessed by an automated system and rejects the request.
- test failures on `test-basics.R` lines 137, 138, 149, `family` comparison: this is caused by 1497e0e9cb022278, where I modified the `$initialize` component of some families - so that saved fits wouldn't match - *but* - these comparisons are supposed to be made by setting the `$initialize` component to NULL for both objects, see [here](https://github.com/glmmTMB/glmmTMB/blob/master/glmmTMB/tests/testthat/test-basics.R#L6-L16). Not sure why it would work with 4.3.3 but not the other versions ... ?

**Comment by bbolker** [comment]
I just put in a bit of code to skip the problematic (test-basics lines 137, 138, 149) if R version is < 4.3.3 ... re-running "allOS" check [here](https://github.com/glmmTMB/glmmTMB/actions/runs/8316290016), we'll see how it goes.

Also restarted my revdep check run after the priors PR merge (thanks!) - will see how it goes.

**Comment by bbolker** [comment]
running revdepcheck locally the only thing I see failing is `sdmTMB`.  I will dig in a bit more ...

(in the meantime it looks like I finally got the test-skipping stuff right, see [here](https://github.com/glmmTMB/glmmTMB/actions/runs/8316540364) ...)

```r
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
   [1] 0.14953175  - 0.14953169  [1]             
   [2] 0.23286508  - 0.23286503  [2]             
   [3] 0.05429365  - 0.05429360  [3]             
   [4] 0.48286508  - 0.48286503  [4]             
   [5] -0.01713492 - -0.01713497 [5]             
   [6] -0.37427778 - -0.37427783 [6]             
   [7] -0.01713492 - -0.01713497 [7]             
   [8] 0.14953175  - 0.14953169  [8]             
   [9] -0.51713492 - -0.51713497 [9]             
  [10] 0.08286508  - 0.08286503  [10]            
   ... ...           ...         and 290 more ...
```

**Comment by mebrooks** [comment]
That sounds better than the revdepcheck results I got from the cluster. I couldn't run it on my computer before because my computer kept freezing and complaining about using too much memory, even when I told the function to only use 2 out of 8 nodes. Are you using a laptop?

I sent the latest version to winbuilder and revdepcheck on the cluster again and should have results soon.

**Comment by bbolker** [comment]
I'm using a fairly powerful machine that I have at home, built near the beginning of the pandemic - 32 cores, 60+GB of memory.  It took about 1 hour to run `revdepcheck` on 26 cores.

**Comment by mebrooks** [comment]
There are still the [3 errors on R-devel according to win-builder](https://win-builder.r-project.org/9I5WG5R8WJg3). I don't think I can work on this much more today, but I can tomorrow.

**Comment by bbolker** [comment]
Those errors seem extremely cryptic to me. Wrote to `r-package-devel@r-project.org` asking for help. Otherwise I don't see any way forward with these, I would say that we should submit the current version and explain to the CRAN maintainers that the checks fail in only one place (win-builder/r-devel), and succeed on (nominally) a different instance of the same platform (i.e., on GitHub actions) - these could well be some kind of obscure false-positive/platform problem ...

**Comment by bbolker** [comment]
Digging down a little bit further into the [`.Rout` file for examples on winbuilder/r-devel](https://win-builder.r-project.org/etH0mJfcXW3F/examples_and_tests/glmmTMB-Ex.Rout) I see that winbuilder/r-devel apparently has Matrix version 1.7.0 installed, which seems weird since CRAN is still listing version 1.6.5 ... ???

```
> ### * </HEADER>
> library('glmmTMB')
Warning in checkMatrixPackageVersion(getOption("TMB.check.Matrix", TRUE)) :
  Package version inconsistency detected.
TMB was built with Matrix version 1.6.5
Current Matrix version is 1.7.0
Please re-install 'TMB' from source using install.packages('TMB', type = 'source') or ask CRAN for a binary version of 'TMB' matching CRAN's 'Matrix' package
> 
```

The [install logs](https://win-builder.r-project.org/etH0mJfcXW3F/00install.out) (scroll to the end) show the same issue.

I'm not sure that's the problem, but it's highly suspicious since the log also shows that the failure occurs on the first non-trivial `glmmTMB()` run in the examples ...

I also suspect that the tests are failing/hanging on `test-Anova.R` because that would also be the first non-trivial example ... (see listing ordering [here](https://win-builder.r-project.org/etH0mJfcXW3F/examples_and_tests/tests/testthat/))

Finally, the `priors` vignette could be the first/only one to fail because (IIRC) all other vignettes are set to not evaluate code chunks on CRAN ..

**Comment by jaganmn** [comment]
The issue is that _we_ **Matrix** maintainers checked **Matrix** 1.7-0 on win-builder.  The checked version became the installed version for everyone else.  It is a "feature", as documented at https://win-builder.r-project.org/:

> Additionally it is possible to install packages serially yourself by uploading them serially: The first package to be uploaded should be the one that is needed by any other packages you upload. Packages you installed yourself are deleted on a regular basis.

So you could get things working again by uploading a **TMB** tarball to win-builder, with the side effect of having **TMB** rebuilt under the currently installed version of **Matrix** (whatever it is at the time).

**Comment by jaganmn** [comment]
(I don't want to step on your toes so I'm just going to let you do one of the above ... Going forward we'll have to remember to clean up after our own checks ... @mmaechler)

**Comment by mebrooks** [comment]
Now it passes checks on R-devel. So uploading the right Matrix version fixed that problem. 

Now I can't understand why, but it doesn't pass `check --as-cran` on my computer
```
* checking whether package ‘glmmTMB’ can be installed ... [150s/151s] WARNING
Found the following significant warnings:
  /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/TMB/include/Eigen/src/Core/arch/NEON/PacketMath.h:1671:38: warning: ‘void* memcpy(void*, const void*, size_t)’ copying an object of non-trivial type ‘Eigen::internal::Packet4c’ {aka ‘struct Eigen::internal::eigen_packet_wrapper<int, 2>’} from an array of ‘const int8_t’ {aka ‘const signed char’} [-Wclass-memaccess]
  /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/TMB/include/Eigen/src/Core/arch/NEON/PacketMath.h:1716:38: warning: ‘void* memcpy(void*, const void*, size_t)’ copying an object of non-trivial type ‘Eigen::internal::Packet4c’ {aka ‘struct Eigen::internal::eigen_packet_wrapper<int, 2>’} from an array of ‘const int8_t’ {aka ‘const signed char’} [-Wclass-memaccess]
See ‘/Users/molliebrooks/Documents/glmmTMB development/glmmTMB/glmmTMB.Rcheck/00install.out’ for details.
* used C++ compiler: ‘g++ (GCC) 11.0.0 20201205 (experimental)’
```

**Comment by bbolker** [comment]
When I ran this with the development version of TMB I got an (expected) WARNING about "Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++);Object: ‘glmmTMB.o’", but nothing else.

I think you should go ahead and submit!  https://en.wiktionary.org/wiki/damn_the_torpedoes

**Comment by mebrooks** [comment]
After I compiled with CRAN's `TMB`, then it passes on my computer again, so I submitted it to CRAN. Finger's crossed. 

I guess we should already start planning for 1.1.10 to coincide with the next `Matrix` release. We can get `dispRE` ready by then.

**Comment by bbolker** [comment]
`foghorn::cran_incoming("glmmTMB")` says

```
  package version    cran_folder time                size 
1 glmmTMB 1.1.9      waiting     2024-03-18 16:04:00 4.2M 
```

according to the `cran_incoming` docs, "waiting" means " waiting for a response from the package maintainer" ... let me know if there's anything I can do.

**Comment by mebrooks** [comment]
Thanks for asking. I'm busy right now with another project. There were 2 failed tests on the insight package, a reverse dependency. I haven't had a chance to look at them yet, but maybe it's also related to the Gaussian sigma change.
```
 ══ Failed tests ════════════════════════════════════════════════════════════════
   ── Failure ('test-get_variance.R:371:3'): fixed effects variance for rank-deficient models, #765 ──
   `out` (`actual`) not equal to 627.03661 (`expected`).

     `actual`: 831
   `expected`: 627
   ── Failure ('test-glmmTMB.R:898:3'): get_paramaters ────────────────────────────
   get_parameters(m0)$Estimate (`actual`) not equal to c(200.03431, -99.71491, 3.20287, 1.38648) (`expected`).

     `actual`: 200.034 -99.715 1.601 0.693
   `expected`: 200.034 -99.715 3.203 1.386
```

**Comment by bbolker** [comment]
Sigh. Seems likely.  I'll work on this if I get a chance.

**Comment by bbolker** [comment]
I fixed this and submitted a PR ([here](https://github.com/easystats/insight/pull/861)), so it's probably OK to go back to CRAN and tell them this is resolved (you could wait until the `insight` maintainers respond to the PR if you want to be safe)

**Comment by strengejacke** [comment]
This _may_ also break code in _performance_ (but I'm planning a new release the next days anyway). I think functions like `icc()` or `r2_nakagawa()` should be affected, if variance components change. On the other hand, if the breaking change is:

> account for the fact that Gaussian dispersion is parameterized as log-SD rather than log-variance now

then tests shouldn't probably be changed, but rather we want to fix `insight::get_variance()`, so that indeed the variance, not SD, is returned again? I think that makes more sense, in order to get correct results for r2/icc again.

**Comment by mebrooks** [comment]
Success on CRAN!
